### DISCUSSION TITLE
Pass Through options changed and docs are not updated to reflect the change

### DISCUSSION THREAD

**Discussion by milanio** [primefaces/community#1085] [PrimeVue]
Hey guys, I believe the docs regarding Pass Through are rather confusing and they do not reflect recent changes. 

https://primevue.org/inputnumber/#pt.demo

I believe some time between versions 3.29.2 - 3.47.2 the way how you pass Pass Through options have changed. Now you need to list attributes you want to pass through to an html element inside the 'root' field. Which I think should be classified as a **breaking change** and should be clearly documented. Or did I miss something?

For InputNumber it used to be : :pt= `{ input: { attr1: 'value1', attr2:'value2' } }` now you need `{ input: { root: { attr1: 'value1', attr2:'value2' } } }`

You can only deduce what's going on by looking at the working example in the bottom of the page. The docs above never mention that there is an option 'root' inside 'input'. Rather it refers to it as 'props'. 

Example:
We need a numeric keyboard to be activated on mobile devices for certain InputNumbers - the code used to work perfectly fine with PrimeVue 3.29.2. It stopped working once PrimeVue was upgraded to 3.47.2

Old way:

```html
 <InputNumber class="p-inputtext-sm" v-model="regularPrice" :pt="{
          input: {
              class: 'p-shorter-input',
              inputmode: 'decimal'
          }
        }" />
```

New way:
```html
 <InputNumber class="p-inputtext-sm" v-model="regularPrice" :pt="{
          input: { 
            root: {
              class: 'p-shorter-input',
              inputmode: 'decimal'
           }
          }
        }" />
```

**Comment by tugcekucukoglu** [comment]
Such changes may occur according to community feedback or needs within PrimeVue itself.

I appreciate your understanding.

**Reply by milanio** [reply]
I don't dispute that, however, creating a breaking change without properly documenting it is less than optimal, wouldn't you agree? Can you make a note in docs for others not to waste their time try to troubleshoot what changed?
